app/views/users/_meta.html.erb

```diff
@@ -1,19 +1,19 @@
 <link rel="canonical" href="<%= user_url(@user) %>" />
-<meta name="description" content="<%= @user.summary %>">
+<meta name="description" content="<%= @user.profile.summary %>">
```

Comment:
All 3 changes in this file were done for the same reason as https://github.com/forem/forem/pull/14210/files#r669313889